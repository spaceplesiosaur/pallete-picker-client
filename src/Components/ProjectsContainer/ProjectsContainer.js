import React, { useState, useEffect } from 'react'
import ProjectsForm from '../ProjectsForm/ProjectsForm'
import ProjectFactory from '../ProjectFactory/ProjectFactory'
import { getData } from '../../apiCalls.js'
import './ProjectsContainer.scss'

const ProjectsContainer = () => {


  return (
    <section className="project-info-page">
      <section className="projectsForm">
        <ProjectsForm />
      </section>
      <section className="projectsList">
        <ProjectFactory />
      </section>
    </section>
  )

}

export default ProjectsContainer
