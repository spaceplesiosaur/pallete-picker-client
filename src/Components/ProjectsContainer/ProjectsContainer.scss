.project-info-page {
    width: 50%;
    margin: 20px;
    height: 100vh;
    border-left: 3px solid #DC65C6;
}