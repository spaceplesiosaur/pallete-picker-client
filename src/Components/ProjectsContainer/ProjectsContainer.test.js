import React from 'react';
import { shallow } from 'enzyme';
import ProjectsContainer from './ProjectsContainer'

describe('ProjectsContainer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ProjectsContainer />)

  })

  it('should render with the correct properites', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
