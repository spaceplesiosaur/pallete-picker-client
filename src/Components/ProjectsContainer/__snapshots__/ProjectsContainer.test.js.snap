// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectsContainer should render with the correct properites 1`] = `
<section
  className="project-info-page"
>
  <section
    className="projectsForm"
  >
    <ProjectsForm
      addNewProject={[Function]}
    />
  </section>
  <section
    className="projectsList"
  >
    <ProjectFactory
      palettes={Array []}
      projects={Array []}
      removeProject={[Function]}
    />
  </section>
</section>
`;
